students/rodicaf/src/fibonacci.ts

```diff
@@ -0,0 +1,31 @@
+export const fibrec  = (n:number) : number => {
+  if ( n <= 1) {
+    return n
+  } else {
+    return fibrec(n-1) + fibrec(n-2)  
+  }
+}; 
+console.time('Time of the fibrec')
+let res = fibrec(2);
+console.timeEnd('Time of the fibrec')
+// console.log(res);
+
+
+export const fibit = (n:number) : number => {
+    // Define an array to store Fibonacci numbers
+    let seq = new Array(n+1); 
```

Comment:
seq is an array that is never reassigned. You can use const. 
Furthermore you could build de solution using two number variables, instead of an array, reducing the `space complexity`